README.md

```diff
@@ -1,3 +1,80 @@
+
+### 2. Ownership
+
+We agree to examine our relationship with ownership, and understand the consequences of our decisions with regard to ownership.
+
+#### 2.a Equal Ownership
+
+We acknowledge that an imbalance in ownership creates an imbalance in the health of our people and our works.
+
+We understand that private ownership is a great responsibility that requires special attention to detail in order to operate sustainably. We have considered releasing our work to the public domain and have determined that private ownership is a more appropriate ownership model for our work.
+
+If we chose to instill a concept of ownership for the project, we agree to share the ownership equally amongst those that maintain and contribute to the project.
+
+#### 2.b Stewardship
+
+As an alternative to private ownership, we have considered releasing our work to the public domain.
+
+We understand that the public domain is a dedicated space, where there is no concept of private ownership. In the event we release our project to the public domain, we agree to serve needs of the general public and act as responsible software stewards. We understand that we have an obligation to the public conservation of our work. As a steward we understand we are in position of burden; and not that of authority.
+
+### 3. Violence and Authoritarianism
+
+We unequivocally condone violence and authoritarianism in any form.
+
+All violent and authoritarian entities are forbidden from our works to the fullest extent permissible by our ownership model and law.
+
+If the works are privately owned, we forbid any violent or authoritarian entities from using the software, contributing to the software, or selling the software. 
+
+If the works are public domain, we forbid any violent or authoritarian entity from contributing to our stewardship of the software. We agree that any violent or authoritarian entity can not serve the needs of the general public.
+
+The list includes (but is not limited to):
+ 
+ - Police and Law enforcement organizations
+ - Military organizations
+ - Totalitarian and authoritarian government
+ - Offensive military operations
+ - Anti-human-rights organizations, or their affiliates
+ - Pro-gun, firearm, or ballistics organizations
+ - Tactical equipment and weapons distributors, resellers, manufacturers, and trading bodies
+ - Affiliates contributing to the success of any violent or authoritarian entity
+
+### 4. Human Rights 
+
+We agree that all lesbian, gay, bisexual, transgender, queer and sexual identity, expression, or orientation should be protected by the fullest extent of law, culture, and legitimacy possible.
+
+We agree that all races, and ethnic backgrounds should have equal rights, experience, and psychological safety of mind.
+
+All entities, individuals, organizations, and contributors who have demonstrated belief or activism otherwise, will be forbidden from our project. This includes but is not limited to:
```

Comment:
Similar to the above, we have 4 items of specificity for The Alphabet but only one specific to racism. I think we should call out examples of racism, just like we call out anti-trans practices separately from anti-lgbtq+ organizations, even though the former would fall under the latter.